Libraries/ReactIOS/NativeMethodsMixin.js

```diff
@@ -37,8 +37,38 @@ type MeasureLayoutOnSuccessCallback = (
   height: number
 ) => void
 
+/**
+ * NativeMethodsMixin provides methods that bypass the React rendering process
+ * and are passed through directly to the underlying native component. This
+ * allows side effects to be performed and actions to be taken that don't fit
+ * into the React model directly.
+ *
+ * The methods described here are available on most of the default components
+ * provided by React Native, and components that extend them. Note however that
+ * they are *not* available on composite components that aren't directly backed
+ * by a native view. This will generally include most components that you define
+ * in your own app. For more information, see [Direct
+ * Manipulation](/react-native/docs/direct-manipulation.html).
+ */
 
```

Comment:
Huh yeah I had tested it with local rendering using the instructions you mentioned, and it seemed to be formatted ok on my machine even with that newline. But whatever, glad it works now!